src/summary/documentation/dimensions-card.resource.ts

```diff
@@ -0,0 +1,51 @@
+import { openmrsObservableFetch } from "@openmrs/esm-api";
+import { forkJoin } from "rxjs";
+import { map } from "rxjs/operators";
+import { Dimensions } from "./dimensions";
+
+const HEIGHT_CONCEPT = "5090AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA";
+const WEIGHT_CONCEPT = "5089AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA";
+
+export function getDimensions(patientId: string) {
+  return forkJoin({
+    weights: getDimensionObservations(patientId, "weight"),
+    heights: getDimensionObservations(patientId, "height")
+  }).pipe(map(data => formatDimensions(data.weights, data.heights)));
+}
+
+function getDimensionObservations(
+  patientId: string,
+  type: "weight" | "height"
+) {
+  const concept = type === "weight" ? WEIGHT_CONCEPT : HEIGHT_CONCEPT;
+  return openmrsObservableFetch(`/ws/fhir/Observation?subject:
```

Comment:
Template strings are whitespace sensitive. This means that your string here has a new line and a bunch of spaces in it. URLS should not have new lines and spaces in them. Let's change this to all one line.

Reply:
the linter does this.